plugins/AudioFileProcessor/AudioFileProcessorWaveView.cpp

```diff
@@ -480,7 +480,10 @@ void AudioFileProcessorWaveView::reverse()
 
 void AudioFileProcessorWaveView::updateCursor(QMouseEvent * me)
 {
-	const auto pos = position(me);
+	QPoint pos;
+	if (me != nullptr) {
```

Comment:
```suggestion
	if (me) {
```
This block can be single lined.

Reply:
I feel like checking for nullptr explicitly would be more clear, but since the C++ guidelines seem to do the same, I won't argue with that